conf/messages.en

```diff
@@ -46,35 +46,43 @@ answersAreInvalid.button = Register for the Economic Crime Levy
 
 start.title = Register for the Economic Crime Levy
 start.heading = Register for the Economic Crime Levy
-start.p1 = Use this service if you are an organisation or entity registered with HMRC for Anti-Money Laundering (AML) supervision and you need to register for the Economic Crime Levy (ECL).
+start.p1 = Use this service to register for the Economic Crime Levy (ECL) if you are an organisation or entity:
+start.b1 = regulated by HMRC for anti-money laundering (AML) supervision
+start.b2 = regulated by one of the Professional Body Supervisors (PBS) for AML supervision
 start.p2 = Check if you {0}.
 start.p2.link = <a href="https://www.gov.uk/government/publications/economic-crime-anti-money-laundering-levy/economic-crime-anti-money-laundering-levy" class="govuk-link">need to pay the levy</a>
 start.p3 = You will be able to submit a return and pay the levy after you have registered. You must pay the levy by 30 September {0}. You will be charged interest if you make a late payment.
 start.beforeYouStart = Before you start
-start.provide = You will need to provide:
-start.provide.b1 = information about your UK revenue for the last financial year
-start.provide.b2 = the date when your organisation started AML-regulated activities, if you started in the past financial year
-start.provide.b3 = details about your organisation so we can confirm your identity
-start.provide.b4 = the details of up to 2 people in your organisation who can act as a point of contact for any activities related to the ECL
+start.provide = You will need to tell us:
+start.provide.b1 = about your accounting period for your organisation - you may have more than one accounting period ending in the financial year
+start.provide.b2 = about your organisation''s UK revenue during this accounting period
+start.provide.b3 = if your company carried out AML-regulated activity for the full financial year
```

Comment:
this doesn't quite align with the prototype - I believe confluence is wrong/out of date in this instance